### DISCUSSION TITLE
Populate all pages with default value for new field

### DISCUSSION THREAD

**Discussion by petemolinero** [statamic/cms#7386] [Q&A]
I'm thinking through the impact of adding fields to a blueprint on an existing site, and have a question. Here's a contrived scenario:

Let's say I have a `pages` collection with 100 entries in it. Then one day I decide I want to add a `rating` text field to the `pages` blueprint; it will have a default value of `3/5`. So I add the field, and I add `{{ rating }}` to my `pages` template to display the rating. I go to my pages, and nothing is rendered—because none of the pages have a value for `rating` yet.

Now, if I go in and edit a page, I see that the default `3/5` is populated in the field—so if I save it, the page will now have the default value. Makes sense.

But, is there a simple way to propagate default values to all entries after creating a new field without having to go and save them all one at a time?

**Comment by jasonvarga** [comment]
Yup.

https://statamic.dev/collections#inject
